coalib/misc/Caching.py

```diff
@@ -14,8 +14,11 @@ class FileCache:
 
     >>> from pyprint.NullPrinter import NullPrinter
     >>> from coalib.output.printers.LogPrinter import LogPrinter
+    >>> from coalib.output.Logging import configure_logging
     >>> import logging
     >>> import copy, time
+    >>> configure_logging()
+    20
     >>> log_printer = LogPrinter()
     >>> log_printer.log_level = logging.CRITICAL
```

Comment:
We definitely need to explain why we `configure_logging` and then use a `LogPrinter`, this is definitely confusing for unaware people^^